yellowstone-grpc-geyser/src/plugin.rs

```diff
@@ -75,6 +77,10 @@ impl GeyserPlugin for Plugin {
             .build()
             .map_err(|error| GeyserPluginError::Custom(Box::new(error)))?;
 
+        // Monitor node health
+        let rpc_client = RpcClient::new("http://localhost:8899".to_string());
```

Comment:
I did not make this an env variable because of speed. I want to get this deployed asap to fix customer impact. I am a RPC land rookie and don't even know how to configure env variables.